up-l3/usubscription/v2/README.adoc

```diff
@@ -218,7 +218,9 @@ image::unsub_local.svg[Unsubscribe Local Flow]
 .Remote Unsubscribe Flow
 image::unsub_remote.svg[Unsubscribe Remote Flow]
 
-* uSubscription *MUST* change the subscriber to itself (core.usubscription) when unsubscribing to remote topics 
+* uSubscription *MUST* change the subscriber to itself (core.usubscription) when unsubscribing to remote topics
```

Comment:
this is expensive copying, an it is an anti-pattern to change the payload.
The system should know - or another communication message needs to be created, meaning it is another message with a **different ID**.
I believe messages should be immutable.
If this is the case, we might have to add a tracking attribute since the message id is not longer something valid we can use to track messages through the system.